openpype/hosts/maya/plugins/publish/extract_camera_mayaScene.py

```diff
@@ -165,7 +164,8 @@ def process(self, instance):
                                                long=True)
 
                         members = members + baked_camera_shapes
-                        members.remove(camera)
+                        for camera in cameras:
+                            members.remove(camera)
```

Comment:
Depending on the amount of cameras you might be expecting this could be quite a slow iteration.

This could then be much faster IF cameras would be a `set`
```members = [member for member in members if member not in cameras]
```

Or:
```python
members = members - cameras
```
If both are sets.